.gitignore

```diff
@@ -4,3 +4,4 @@ composer.lock
 .php_cs.cache
 .DS_Store
 Thumbs.db
+.idea/
```

Comment:
Why not? It hurts nothing... Only is useful for people who use a certain IDE, just like how there are ignores intended for people who use a certain OS, etc.